src/factories/ethers-interface.ts

```diff
@@ -32,3 +28,37 @@ export async function ethersInterfaceFromSpec(spec: FakeContractSpec): Promise<e
     return new ethers.utils.Interface(foundInterface);
   }
 }
+
+async function ethersInterfaceFromAbi(abi: string): Promise<ethers.utils.Interface> {
+  try {
+    return new ethers.utils.Interface(abi);
+  } catch (err) {
```

Comment:
Does `catch (err: Error)` work here instead?

Reply:
Type annotation for the catch clause variable is restricted, TS throws this error:
```Catch clause variable type annotation must be 'any' or 'unknown' if specified.ts(1196)```